packages/components/src/Tabs/Tabs.tsx

```diff
@@ -52,7 +52,7 @@ export function Tabs({ children, defaultTab = 0, onTabChange }: TabsProps) {
 
   const activeTabProps = (React.Children.toArray(children) as ReactElement[])[
     activeTab
-  ].props;
+  ]?.props;
 
   const handleOverflowing = () => {
     if (tabRow.current) {
```

Comment:
@ZakaryH Where does tabRow.current get set? I see it gets passed to a `ul` element.

Reply:
that's actually where it is being set!

```
<ul ... ref={tabRow}>
```

is basically shorthand for

```
<ul ... ref={(ulRef) => tabRow.current = ulRef}>
```

Reply:
Ah neato! Thanks!